functests/5_latency_testing/latency_testing.go

```diff
@@ -35,6 +35,7 @@ const (
 	incorrectMsgPart2                  = " has incorrect value"
 	invalidNumber                      = " has an invalid number"
 	maxInt                             = "2147483647"
+	minimumCpuForOslat                 = "2"
```

Comment:
If you have similar variables in multiple packages, you can move them into a common package and import them from there.
But this can be done on a separate PR.

Reply:
good idea, I'll consider that next. Thanks!